libethash-cuda/keccak.cuh

```diff
@@ -784,7 +784,11 @@ DEV_INLINE void SHA3_512(uint2* s)
         s[24] = chi(s[24], u, v);
 
         /* iota: a[0,0] ^= round constant */
+#if __CUDA_ARCH__ >= 320
```

Comment:
As with `SHFL`, can you make a macro `LDG` that hides the beauty of the `#if #else #endif`?

Reply:
fixed and rebased~